src/lib/openapi/spec/integration-events-schema.ts

```diff
@@ -10,15 +10,7 @@ export const integrationEventsSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['integrationEvents'],
-    properties: {
-        integrationEvents: {
-            type: 'array',
-            description: 'A list of integration events.',
-            items: {
-                $ref: integrationEventSchema.$id,
-            },
-        },
-    },
+    properties: {},
```

Comment:
Hmmm this doesn't seem right if we're keeping the feature and just removing the feature flag

Reply:
I'm guessing this is because the did a grep for `integrationEvents` without considering whether something is a string or a "name".

Reply:
We do not do "grep" to transform the code. There are scenarios where it is OK to delete this entry, e.g., 
```
flags: {
            integrationEvents: true,
        },
      ```
      We need to discuss and agree on when it is safe to delete and when it is not. @sjaanus